programs/bench.c

```diff
@@ -41,6 +41,7 @@
 #include <string.h>      /* memset */
 #include <stdio.h>       /* fprintf, fopen, ftello */
 #include <time.h>        /* clock_t, clock, CLOCKS_PER_SEC */
+#include <assert.h>      /* assert */
```

Comment:
Does lz4 have a `LZ4_DEBUG=X` parameter, or does it set `-DNDEBUG` for optimized builds?

Reply:
lz4 has a `LZ4_DEBUG=X` parameter, but that's only for `/lib`.

Modules in `/programs` just enable `assert()` all the time.
That's because they are not performance sensitive.
Even in `bench.c`, `assert()` is used in few places and outside of timing loops.